adala/runtimes/base.py

```diff
@@ -12,6 +12,31 @@
 tqdm.pandas()
 
 
+class CostEstimate(BaseModel):
```

Comment:
why not using a unique type defined in SDK, here and in LSE?

Reply:
We don't want to depend on the sdk anywhere other than the LSE-specific result handler.
We were told that adala is meant to be runnable separately from LSE, but lmk if we're planning to couple them together now and we can start doing things like this.

Reply:
Well, it is about LSE if we have to add LSE as a dependency. Are you not proposing that the LSE SDK becomes a dependency of adala? (outside of the LSE handler, which is the only place we currently depend on the SDK)